wandb/sdk/wandb_init.py

```diff
@@ -446,18 +448,87 @@ def init(
     id=None,
     settings: Union[Settings, Dict[str, Any], None] = None,
 ) -> RunBase:
-    """Initialize a wandb Run.
+    """Initialize W&B
+    Spawns a new process to start or resume a run locally and communicate with a
+    wandb server. Should be called before any calls to wandb.log.
 
     Args:
-        entity: alias for team.
-        team: personal user or team to use for Run.
-        project: project name for the Run.
+        job_type (str, optional): The type of job running, defaults to 'train'
+        dir (str, optional): An absolute path to a directory where metadata will
+            be stored.
+        config (dict, argparse, or absl.flags, str, optional):
+            Sets the config parameters (typically hyperparameters) to store with the
+            run. See also wandb.config.
+            If dict, argparse or absl.flags: will load the key value pairs into
+                the runs config object.
+            If str: will look for a yaml file that includes config parameters and
+                load them into the run's config object.
+        project (str, optional): W&B Project.
+        entity (str, optional): W&B Entity.
+        reinit (bool, optional): Allow multiple calls to init in the same process.
+        tags (list, optional): A list of tags to apply to the run.
+        group (str, optional): A unique string shared by all runs in a given group.
+        name (str, optional): A display name for the run which does not have to be
+            unique.
+        notes (str, optional): A multiline string associated with the run.
+        magic (bool, dict, or str, optional): magic configuration as bool, dict,
+            json string, yaml filename.
+        config_exclude_keys (list, optional): string keys to exclude storing in W&B
+            when specifying config.
+        config_include_keys (list, optional): string keys to include storing in W&B
+            when specifying config.
+        anonymous (str, optional): Can be "allow", "must", or "never". Controls
+            whether anonymous logging is allowed.  Defaults to never.
+        mode (str, optional): Can be "online", "offline" or "disabled". Defaults to
+            online.
+        allow_val_change (bool, optional): allow config values to be changed after
+            setting. Defaults to true in jupyter and false otherwise.
+        resume (bool, str, optional): Sets the resuming behavior. Should be one of:
+            "allow", "must", "never", or "auto". Defaults to "never".
```

Comment:
It actually defaults to `None`, which won't crash if an existing run exists.  @raubitsj what do you think about making the default "never"?  It seems like that would prevent accidentally overwriting an old run.

Reply:
is none the same as auto?

Reply:
None means don't resume, just overwrite the run if it exists.